scripts/settings.js

```diff
@@ -31,7 +32,15 @@ export let debug = false;
  * The user-selected display currency from market-aggregator sites
  * @type {string}
  */
-export let strCurrency = 'usd';
+export let strCurrency = getDefaultCurrency();
+
+/**
+ * @returns {string} currency based on settings
+ */
+function getDefaultCurrency() {
+    const langCode = navigator.languages[0]?.split('-')?.at(-1) || 'US';
+    return getCurrencyByAlpha2(langCode).toLowerCase() || 'usd';
+}
```

Comment:
We should ensure this currency is available from CoinGecko (just in case) as I'm not sure CoinGecko support ***every*** possible fiat currency, i.e: use `await cMarket.getCurrencies();` to fetch CoinGecko's available currencies.

Reply:
Done